Makefile

```
@@ -42,6 +43,14 @@ build/chainwatch:
 	cp $(LOTUS_PATH)lotus-chainwatch build/chainwatch
 BINS+=build/chainwatch
 
+.PHONY: visor
+visor: deps build/visor
+
+build/visor:
+	$(MAKE) -C $(VISOR_PATH) deps sentinel-visor
+	cp $(VISOR_PATH)sentinel-visor build/sentinel-visor
+BINS+=build/visor
```

Should this be `sentinel-visor`? It looks like its used by the clean target to delete the binary.

```suggestion
BINS+=build/sentinel-visor
```

---

Good catch and your general comment is spot on. I tried to keep the binary name unchanged from the default but shortened the makefile targets to visor for convenience. I'm not sure if we'll keep sentinel-visor as a long term thing or if it will just be folded into a single sentinel binary

---

I think anything which is in the public namespace should have the `sentinel-` prefix (for distinction purposes primarily) but dev-only interfaces can use the non-prefixed name (Ex: `make visor`) and should abstract away which name should be used at any given time. This approach becomes useless with a case where we can't avoid typing `sentinel-visor` (for ex) as part of some command enough that it's annoying. I can't think of any case like this.

Examples of public namespaces include: linux filesystem service install paths, (2) public repo paths and similar URIs, (3) package system namespaces.
Examples of non-public namespaces include: local build artifacts